### PR TITLE
User story 16 create a new world

### PR DESCRIPTION
**Pull request by Oskmey**
Second World
Skärmbild 2024-05-08 165057

### CODE REVIEW

File: DAT257AgileProject/Assets/Scenes/Game/Second World.unity
Comment (Caramajau) [inline DAT257AgileProject/Assets/Scenes/Game/Second World.unity:1]: Looks good. Though it would probably be nice to have the second map as a prefab.

Comment (Caramajau) [reply DAT257AgileProject/Assets/Scenes/Game/Second World.unity:1]: Maybe it is also possible to make the intended fishing spot a little clearer?

Comment (Oskmey) [reply DAT257AgileProject/Assets/Scenes/Game/Second World.unity:1]: Yes, we considered changing the texture of the fishing spots to accurately reflect that those places are polluted.

Comment (Oskmey) [reply DAT257AgileProject/Assets/Scenes/Game/Second World.unity:1]: Added some texture for fishing spot to make it clearer that you could fish there and made map a prefab with the fishingspots

**Comment by Oskmey** [comment]
Added fishingspot textures

**Review by Caramajau — APPROVED** [review]

**Review by linuskar — CHANGES_REQUESTED** [review]
Add credits for the second world

**Review by linuskar — APPROVED** [review]
ok

**Pull request merged**